config.php.default

```diff
@@ -8,6 +8,13 @@ $config['db_user'] = 'USERNAME';
 $config['db_pass'] = 'PASSWORD';
 $config['db_name'] = 'librenms';
 
+
+// OpenTSDB configuration
```

Comment:
You should move all of this into includes/defaults.inc.php but you must set `$config['opentsdb']['enable'] = false;` as people won't have this setup to use by default.